### DISCUSSION TITLE
GitHub Advanced Security Trial for Enterprise Accounts

### DISCUSSION THREAD

**Discussion by tuves** [community/community#57808] [Enterprise]
### GitHub Advanced Security is now available on GitHub Enterprise Cloud 

**UPDATE**
You can now access CodeQL, Secret Scanning, and other features of GitHub Advanced Security as part of your 30 days GitHub Enterprise Cloud free trial. Enterprise admins can enable GitHub Advanced Security under Enterprise licensing.

![blog_ghas](https://github.com/community/community/assets/49915749/4b628564-1773-4416-b32c-27e8a77b3d2c)

--------------------------------------------------------------
**June 12, 2023**

For 14 days, all eligible GitHub Enterprise Accounts can try GitHub Advanced Security. Enterprise admins can enable the trial under the Enterprise licensing tab and run CodeQL, Secret Scanning, and access Dependency Review across all organizations.

![238361604-dced7dfd-52c2-4502-b468-9f2218db3f6f](https://github.com/community/community/assets/75342807/ffa912ec-e565-46a0-8729-4bc019f48c58)

🔗  [Learn more about GitHub Advanced Security](https://github.com/features/security)

**Comment by Shiboleth2** [comment]
Pls have an admim contact me. Thank you

**Comment by yuchimaggie** [comment]
HI  
We have problems with development tradingview  as follows

 My session setting as follows如下

session: '0845-1345,1500-2400,0000-0501:23456|0000-0501:7|0845-1346,1500-2400:2'

But when getting the bar, the time is as follows, and it doesn't cross the Saturday and Sunday to find last friday's data

So it causes query problems and can't be queried after a week


[getBars]: Method call {ticker: 'TAIFEX:future:MXFF3:0', name: 'MXFF3', description: '小型臺指202306', type: 'future', session: '0845-1345,1500-2400,0000-0501:23456|0000-0501:7|0845-1346,1500-2400:2', …} 1 1685898813 1686023378 Mon Jun 05 2023 01:13:33 GMT+0800 (台北標準時間) Tue Jun 06 2023 11:49:38 GMT+0800 (台北標準時間)
  Mon Jun 05 2023 01:13:33 GMT+0800 (台北標準時間) Tue Jun 06 2023 11:49:38 GMT+0800 (台北標準時間) 1645
localhost?5500/src/datafeed.js:228
data.data.length 1324



[getBars]: Method call {ticker: 'TAIFEX:future:MXFF3:0', name: 'MXFF3', description: '小型臺指202306', type: 'future', session: '0845-1345,1500-2400,0000-0501:23456|0000-0501:7|0845-1346,1500-2400:2', …} 1 1685874506 1685898813 Sun Jun 04 2023 18:28:26 GMT+0800 (Taipei Standard time) Mon Jun 05 2023 01:13:33 GMT+0800 (Taipei Standard time)
  Sun Jun 04 2023 18:28:26 GMT+0800 (台北標準時間) Mon Jun 05 2023 01:13:33 GMT+0800 (Taipei Standard time 321

How can we solve it?
Thanks

**Comment by sanba06c** [comment]
Is it available for the user who is using GitHub Enterprise Cloud trial version? I couldn't find it on Enterprise licensing tab.

**Reply by tblanarik** [reply]
Great question, @sanba06c 
For right now, it’s only available for paid Enterprise Accounts, but we are looking forward to allowing both trials to happen simultaneously, stay tuned!

**Reply by salkasov1999** [reply]
@tblanarik I can't seem to find a normal way to access learning to code! Can you advise where I can shove myself, I’m still from Russia, I don’t know if this is connected or not, you can unsubscribe to me

**Reply by Maxtrade4** [reply]
It is difficult to understand

**Reply by esmitjan5** [reply]
من ناشنوا ونابینا هستم و تقاضای طرح خودکار برای پلتفرم خودرادارم

**Reply by tblanarik** [reply]
@sanba06c - just wanted to revisit this thread and let you know that GitHub Enterprise Cloud Trial users can now also try Advanced Security at the same time! Thanks for your patience!

**Comment by 257811innocent** [comment]
Thanks for your help me plz

**Comment by Mohamad5159** [comment]
چگونه کلیدعمومی ایپی بدست بیارم

**Comment by stndc** [comment]
Thanks!!

**Comment by jamessv** [comment]
warming information, love to see this type of msg again

**Comment by KrenntCraven** [comment]
Thank you!

**Comment by Gitchasum** [comment]
Hi who is this?  Whats this regarding?

**Comment by evan-day** [comment]
Would GitHub ever consider offering GHAS as an add-on for individuals with GitHub Pro, similar to how Copilot is an add-on? I understand the target market is the enterprise & one can get the features for free if their project is open source. Most of my code I keep private, but I would also enjoy being able to use a tool like GHAS and if I liked it, maybe make a case for why it should be used in ones organisation :)
